code/datums/components/twohanded.dm

```diff
@@ -257,9 +257,9 @@
 		parent_item.force -= sharpened_increase
 	if(force_multiplier)
 		parent_item.force /= force_multiplier
-	else if(!isnull(force_unwielded))
+	else if(force_unwielded)
 		parent_item.force = force_unwielded
-	if(!isnull(block_power_unwielded))
+	if(block_power_unwielded)
```

Comment:
The problem with this way is that it stops you from having things that do 0 damage or can't block when unwielded. I think a better solution would be to set force_unwielded to null on the addcomponent, since by default the value is 0

Reply:
Brain smooth, I just copied from what wield() did, I'll change both